src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -2,11 +2,17 @@
 
 public class BinaryString {
 
-    /**
-     * Напишите метод toBinaryString(), который принимает на вход целое число value,
-     * а возвращает String с представлением этого числа в двоичном виде.
-     */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder binaryStr = new StringBuilder();
+        int res = 0;
```

Comment:
can we solve it without creating this variable?

Reply:
Yes, done